packages/subgraph/src/kpi-tokens-manager.ts

```diff
@@ -183,3 +222,92 @@ export function handleUpgradeTemplate(event: UpgradeTemplateEvent): void {
     }
     newTemplate.save();
 }
+
+export function handleSetFeatureSetOwner(event: SetFeatureSetOwnerEvent): void {
+    const oraclesManager = getKPITokensManager();
```

Comment:
Same as above.

Reply:
Updated in 0dedba60ec462b3ca183a04719fe5d88346aa841